src/com_tjvendors/site/includes/utilities.php

```diff
@@ -186,4 +187,94 @@ public function defineLanguageConstant()
 	{
 		Text::script('COM_TJVENDOR_VENDOR_FORM_AJAX_FAIL_ERROR_MESSAGE');
 	}
+
+	/**
+	 * This function returns the vendor country list.
+	 * For e.g in JGive there is 3 vendor belongs from 3 different countries,
+	 * for filtering campaigns by country it returning the array of those 3 countries
+	 *
+	 * @param   string  $extension  extension name like com_jgive
+	 * 
+	 * @return   Array 
+	 *
+	 * @since   __DEPLOY_VERSION__
+	 */
+	public function getFilterCountries($extension)
```

Comment:
@deepgawade89 Rename function to getCountries and pass filter as array. We are writing this in utilities so function should be useful in all the possible conditions